projects/plugins/jetpack/modules/comment-likes.php

```diff
@@ -105,16 +118,22 @@ function enqueue_admin_styles_scripts() {
 				'modules/comment-likes/comment-like-count.js'
 			),
 			array( 'jquery' ),
-			JETPACK__VERSION
+			JETPACK__VERSION,
+			$in_footer = false
 		);
 	}
 
+	/**
+	 * Adds like count column to admin page.
+	 *
+	 * @param array $columns - column of admin table.
+	 * */
```

Comment:
I'm a little surprised phpcs doesn't complain about this.
```suggestion
	 */
```

Looks like you did it in a few other places in here too.